src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
I think we should use it, definitely. The "attempts remaining" element is at least tabbable, which it wasn't before now. So if for some reason the user decides to use Safari, then the element is still accessible in some way. I can't think of any other ways to include it, but I'm new to accessibility. 

I would also suggest that the user should get some feedback from their answer when they click submit on a question component. Currently there is no feedback at all, so if they do get the question correct, a partially sighted user wouldn't even know.